worldengine/draw.py

```diff
@@ -9,6 +10,36 @@
 # Helper values
 # -------------
 
+### For draw_satellite ###
+NOISE_RANGE = 10 # a random value between -NOISE_RANGE and NOISE_RANGE will be added to the rgb of each pixel
+
+# These are arbitrarily-chosen elevation cutoffs for 4 different height levels. 
+# Some color modifiers will be applied at each level
+HIGH_MOUNTAIN_ELEV = 215
```

Comment:
I understand the concept, but I don't really understand how to use the function. Is it documented anywhere? If this function exists, why are you suggesting to use the code you wrote out? What does the `[3]` in that code mean?